source/_posts/2017-05-19 kanban-method

```diff
@@ -0,0 +1,74 @@
+# Kanban method in a consulting company
```

Comment:
Ok, I just edit and update this file, and I will not create PR with another extension.